TLM/TLM/Manager/Impl/TrafficLightManager.cs

```diff
@@ -108,6 +108,13 @@ public class TrafficLightManager
             return SetTrafficLight(nodeId, false, ref node, out reason);
         }
 
+        public void RemoveAllExistingTrafficLights() {
+            Array16<NetNode> nodes = NetManager.instance.m_nodes;
+            for (ushort i = 0; i < nodes.m_size; i++) {
+                RemoveTrafficLight(i, ref nodes.m_buffer[i]);
```

Comment:
I had a very quick look in the code it seems there isn't memory moved on each traffic light removal, so should be fine. If this ever becomes slow, we can optimize it. Just keep an eye on performance here.

Reply:
I will check performance of that method later :) 
This function will be triggered from mod options maintenance tab on demand, after user confirm this operation.